x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/edit_package_policy_page/index.tsx

```diff
@@ -84,6 +104,15 @@ export const EditPackagePolicyForm = memo<{
   const [originalPackagePolicy, setOriginalPackagePolicy] = useState<
     GetOnePackagePolicyResponse['item']
   >();
+  const [dryRunData, setDryRunData] = useState<UpgradePackagePolicyDryRunResponse>();
+  const routeState = useIntraAppState<EditPackagePolicyRouteState>();
```

Comment:
are we setting `routeState.from` using state here? I am not sure I saw it in the PR

Reply:
No - this is leftover from a previous implementation. I'll clean this up.